llvm/test/Transforms/IndVarSimplify/indvars-preserving-debugloc.ll

```diff
@@ -0,0 +1,61 @@
+; RUN: opt < %s -passes=indvars -S | FileCheck %s
```

Comment:
Apologies for another file name nit... Just noticed the file name starts with "indvars-", but it's already in the `IndVarSimplify` directory, so the "indvars-" prefix feels redundant (and most existing tests in that directory do not have it), so perhaps best to remove that here.

Reply:
Thanks for the suggestion!